crates/hyperswitch_connectors/src/connectors/worldline/transformers.rs

```diff
@@ -1,21 +1,25 @@
 use api_models::payments;
-use common_utils::pii::Email;
+use common_enums::enums::{AttemptStatus, BankNames, CaptureMethod, CountryAlpha2, Currency};
+use common_utils::{pii::Email, request::Method};
+use hyperswitch_domain_models::{
+    payment_method_data::{BankRedirectData, PaymentMethodData},
+    router_data::{ConnectorAuthType, RouterData},
+    router_flow_types::{
+        payments::Authorize,
+        refunds::{Execute, RSync},
+    },
+    router_request_types::{PaymentsAuthorizeData, ResponseId},
+    router_response_types::{PaymentsResponseData, RedirectForm, RefundsResponseData},
+    types::{PaymentsAuthorizeRouterData, PaymentsCaptureRouterData, RefundsRouterData},
+};
+use hyperswitch_interfaces::{api::CurrencyUnit, errors};
 use masking::{PeekInterface, Secret};
 use serde::{Deserialize, Serialize};
 use url::Url;
 
 use crate::{
-    connector::utils::{self, CardData, RouterData},
-    core::errors,
-    services,
-    types::{
-        self,
-        api::{self, enums as api_enums},
-        domain,
-        storage::enums,
-        transformers::ForeignFrom,
-        PaymentsAuthorizeData, PaymentsResponseData,
-    },
+    types::{RefundsResponseRouterData, ResponseRouterData},
+    utils::{self, CardData as _, RouterData as _},
```

Comment:
Can we avoid the `as _` imports, or does not having it cause compilation errors?

Reply:
Done